index.js

```diff
@@ -1,11 +1,11 @@
 const debug = require("debug")("apn");
 
 const credentials = require("./lib/credentials")({
-	debug
+	logger: debug
 });
 
 const config = require("./lib/config")({
-	debug,
+	logger: debug,
```

Comment:
In config.js the debug logger is accessed via `dependencies.debug`. Please also adjust config.js to use `logger` instead of `debug`.